Gemfile

```
@@ -142,14 +142,14 @@ group :test do
   gem "capybara", "~> 3.39"
   gem "json-schema"
   gem "rspec_junit_formatter"
-  gem "selenium-webdriver"
   gem "shoulda-matchers"
   gem "webmock"
   gem "simplecov"
   gem "simplecov-lcov"
   gem "undercover"
   gem "undercover-checkstyle"
   gem "percy-capybara"
+  gem "capybara-playwright-driver"
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `capybara-playwright-driver` should appear before `percy-capybara`.

(Bundler/OrderedGems)